### PR TITLE
Disable zeroing workgroup memory for compute shaders.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This is a performance improvement for shader compilation.

See https://github.com/gfx-rs/wgpu/pull/5508

### CODE REVIEW

File: src/wgpu_engine.rs [hunk 1]
```diff
@@ -644,7 +644,10 @@ impl WgpuEngine {
             layout: Some(&compute_pipeline_layout),
             module: &shader_module,
             entry_point: "main",
-            compilation_options: PipelineCompilationOptions::default(),
+            compilation_options: PipelineCompilationOptions {
+                zero_initialize_workgroup_memory: false,
+                ..PipelineCompilationOptions::default()
```
Comment (DasLixou) [inline src/wgpu_engine.rs:649]: Do we have a policy of whether using `..Default::default()` or `..TheWholeLongStructName::default()` ?

Comment (raphlinus) [reply src/wgpu_engine.rs:649]: I personally go for the former.

Also, as a point of curiosity, less noisy default syntax was one of the things discussed at the unconference for how the Rust language could better support UI and application programming.

Comment (DasLixou) [reply src/wgpu_engine.rs:649]: Only thing I know is that there was a [rust internals discussion](https://internals.rust-lang.org/t/add-default-keyword/20810/13) about a `default` keyword like in C#

Comment (DasLixou) [inline src/wgpu_engine.rs:649]:
```suggestion
                ..Default::default()
```

Comment (DasLixou) [reply src/wgpu_engine.rs:649]: oh wait I can also put it here :0 github actually surprises me sometimes
```suggestion
                ..Default::default()
```

**Review by DJMcNab — APPROVED** [review]
I have the `Default::default` version running on my phone with no issues and the expected performance improvement

So once we apply that suggestion, I'm happy for this to land.

**Comment by waywardmonkeys** [comment]
Used the proposed change and set merge when ready as I am in bed to sleep.

**Pull request merged**